dexbot/plugin.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+import threading
+import importlib
+import logging
+
+import dexbot.plugins
+from dexbot.helper import iter_namespace
+
+from bitshares import BitShares
```

Comment:
Works but to be more specific change to `from bitshares.bitshares import BitShares`.